src/components/ActivityFeed/PostItem.tsx

```
@@ -0,0 +1,367 @@
+					<Divider className='m-0 rounded-lg border-[0.6px] border-solid border-[#D2D8E0] p-0 dark:border-[#4B4B4B]' />
+					<PostActions
+						post={post}
+						reactionState={reactionState}
+						setReactionState={setReactionState}
+					/>
+					<PostCommentSection post={post} />
+					{isMobile && (
+						<div
+							onClick={() => {
+								if (currentUserdata && currentUserdata?.id) {
+									setShowModal(true);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)